src/main/java/io/github/jsonSnapshot/SnapshotMatchingStragety.java

```diff
@@ -0,0 +1,7 @@
+package io.github.jsonSnapshot;
+
+public interface SnapshotMatchingStragety {
```

Comment:
Typo in `Strategy`

Reply:
fixed in https://github.com/json-snapshot/json-snapshot.github.io/pull/15/commits/863d2adecadc7d005918e0d152a1a8458d2117fc